hack/loadtest/util/help.go

```diff
@@ -30,6 +31,16 @@ func (i *ClusterNames) Set(value string) error {
 	return nil
 }
 
+func printMetricFamily(mf *io_prometheus_client.MetricFamily) {
+	results := mf.GetName() + " (sec): "
+	for _, m := range mf.GetMetric() {
+		for _, q := range m.GetSummary().GetQuantile() {
+			results += fmt.Sprintf("[Quantile %v: %v] ", q.GetQuantile(), q.GetValue())
+		}
+	}
+	klog.Info(results)
+}
```

Comment:
what's the output look like?

Reply:
`quantile_apply_crp_latency (sec): [Quantile 0.5: 0.505853678] [Quantile 0.9: 0.754912736] [Quantile 0.99: 0.756766946]`